specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -1835,6 +2093,39 @@
       ],
       "description": "The JIT scheduling policies."
     },
+    "JitApprovalMode": {
+      "description": "The Jit approval mode.",
+      "enum": [
+        "NotSpecified",
+        "AutoApprove",
+        "ManualApprove"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "JitApprovalMode",
+        "modelAsString": true
+      }
+    },
+    "JitApproverDefinition": {
+      "properties": {
+        "id": {
```

Comment:
principalId or similar.  "id" is not descriptive enough

Reply:
Discussed offline, will change this in a later version.